### DISCUSSION TITLE
Redash docker setup

### DISCUSSION THREAD

**Discussion by kunalchander** [getredash/redash#6959] [Q&A]
Can anyone provide me with the docker compose for setting up Redash . As I am getting this when i am trying downloading 
![Screenshot 2024-05-10 114437](https://github.com/getredash/redash/assets/100569457/3cd06266-bcc5-4c30-a9df-eecfed1397bc)

**Comment by justinclift** [comment]
Ahhh.  It's been renamed to [compose.yaml](https://github.com/getredash/setup/blob/master/data/compose.yaml).

You're probably using an old version of the setup script.  Unless you *really* want to use that specific version of the setup script for some reason, I'd recommend you grab the very latest version of it:

https://github.com/getredash/setup/raw/master/setup.sh
